### PR TITLE
Split release and release-group disambiguation into separate fields.

### PR DESCRIPTION
**Pull request by djl**
Here's a pretty rough first attempt at fixing #3024.

`releasegroupdisambig` seems needlessly long but it does follow the naming conventions of other fields.

### CODE REVIEW

File: beets/autotag/hooks.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ class AlbumInfo(object):
     - ``albumstatus``: MusicBrainz release status (Official, etc.)
     - ``media``: delivery mechanism (Vinyl, etc.)
     - ``albumdisambig``: MusicBrainz release disambiguation comment
+    - ``releasegroupdisambig``: MusicBrainz release group disambiguation comment
```
Comment (sampsyo) [inline beets/autotag/hooks.py]: Travis has the most nitpicky style nitpick ever on this line: https://github.com/beetbox/beets/pull/3025/files

File: beets/autotag/mb.py [hunk 2]
```diff
@@ -346,13 +346,11 @@ def album_info(release):
     info.releasegroup_id = release['release-group']['id']
     info.albumstatus = release.get('status')
 
-    # Build up the disambiguation string from the release group and release.
```
Comment (sampsyo) [inline beets/autotag/mb.py]: Maybe preserving a version of this comment would be useful; for example:

> Get the disambiguation strings at the release and release group level.

**Comment by djl** [comment]
Addressed the comments and made Travis green again.

**Comment by sampsyo** [comment]
Perfect; thanks!!

**Pull request merged**